src/components/MasterDetails.js

```diff
@@ -103,6 +104,10 @@ type MasterDetailsProps = {
    * The class name to apply to the table element. Optional.
    */
   tableClassName: string;
+  /** Optional background color to apply to the last selected row. Only used if multiSelect is specified as well. Takes precedence
+   *  over all other background colors specified through classNames.
+   */
+  anchorRowBackgroundColor: ?string;
```

Comment:
I'm actually wondering we should use the normal select color for the anchor row and use the extra color for the selected-but-not-showing-in-the-inspector rows... This would mean, for example, that a multi-select table with one row selected would look the same as a single-select t table with one row selected... what do you think?

Reply:
I did it this way to reduce the impact of potentially existing tables. Anyone using the table today won't see a style difference. If we're comfortable with forcing that styling change, I'm fine with that.

Reply:
Can we also know that no customers are using this? Here, the change is pretty minor and the impact as well so I'll go ahead and change it here, but generally speaking, are we concerned about the impact outside of the organization?